cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java

```diff
@@ -185,16 +181,6 @@ private static void printBallerinaDistPath() {
         }
     }
 
-    private static String getMessageForInternalErrors() {
-        String errorMsg;
-        try {
-            errorMsg = BCompileUtil.readFileAsString("cli-help/internal-error-message.txt");
```

Comment:
Shall we remove the `internal-error-message.txt` file as well? I think it is not used in other places.

Reply:
Removed it.